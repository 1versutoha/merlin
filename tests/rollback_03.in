["tell","struct","module M = struct\n  let uniq = 42\nend\nopen M\nlet _ = (uniq\n"]
["tell","end",null]
["type","enclosing",{"line":5,"col":11}]
