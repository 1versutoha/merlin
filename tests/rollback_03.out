["return",false]
["return",true]
["return",[2,[{"start":{"line":5,"col":8},"end":{"line":5,"col":13},"type":"int"}]]]
